metricbeat/module/kibana/_meta/config.yml

```diff
@@ -3,5 +3,6 @@
   #  - status
   period: 10s
   hosts: ["localhost:5601"]
+  #basepath: ""
```

Comment:
Is that a config that non dev users will use? If yes, is it common? If not, lets skip it in the short config.